Landing CV/index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Landing CV</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+    <div class="container-head">
+        <header>
+            <ul class="nav-items">
+                <li><a href="#">About</a></li>
+                <li><a href="#">Skills</a></li>
+                <li><a href="#">Experience</a></li>
+                <li><a href="#">Project</a></li>
+                <li><a href="#">Educational</a></li>
+                <li><a href="#">Languages</a></li>
+                <li><a href="#">Contact me</a></li>
+            </ul>
+        </header>
+        <hr>
+        <article>
+            <div class="article-about">
+                <h1>Sergei Alyoshin</h1>
+                <p class="article-about-title">Front-end developer</p>
+                <p class="article-about-description">Hi, I am Sergei Alyoshin and I am Junior Front-end<br>
+                    developer.<br>
+                    I have completed Front-end courses at IT School Intellect<br><br>
+                    Fox and have basic knowledge of HTML, CSS, JavaScript,<br>
+                    React and Redux.</p>
+                <div class="article-button">
+                    <button class="button-download">Download CV</button>
+                    <button class="button-contact">Contact me</button>
+                </div>
+            </div>
+            <img src="images/photo.svg" alt="photo">
+        </article>
+    </div>
+        <section class="suggestions">
+            <div class="suggestions--heading">
+                <p class="heading">Skills</p>
+            </div>
+            <article class="row">
+                <figure class="suggestions__col">
+                    <img class="suggestions__img-wrap" src="images/HTML.svg" alt="html">
+                </figure>
+                <figure class="suggestions__col">
+                    <img class="suggestions__img-wrap" src="images/CSS.svg" alt="html">
+                </figure>
+                <figure class="suggestions__col">
+                    <img class="suggestions__img-wrap" src="images/SASS.svg" alt="html">
+                </figure>
+                <figure class="suggestions__col">
+                    <img class="suggestions__img-wrap" src="images/GIT.svg" alt="html">
+                </figure>
+                <figure class="suggestions__col">
+                    <img class="suggestions__img-wrap" src="images/JavaScript.svg" alt="html">
+                </figure>
+                <figure class="suggestions__col">
+                    <img class="suggestions__img-wrap" src="images/Github.svg" alt="html">
+                </figure>
+            </article>
```

Comment:
it should be done as img)
images can be icons and stars. everything else should be written by code.
Also use different alt for all skils